src/cli.rs

```diff
@@ -40,25 +39,30 @@
 //!     pub name: String,
 //! }
 //!
+//! #[derive(clap::Parser)]
+//! #[clap(
+//!     name = "Spark Operator",
+//!     author,
+//!     version,
+//!     about = "Stackable Operator for Foobar"
```

Comment:
"Stackable Operator for Spark"

Reply:
Switched the remaining names to "Foobar Operator".